app/helpers/application_helper.rb

```diff
@@ -120,7 +120,10 @@ def get_icon_text(icon, text)
   def image_link_to(image_path, url, image_tag_options = {},
                     link_to_options = {})
     link_to url, link_to_options do
-      image_tag image_path, image_tag_options
+      default_alt = File.basename(image_path, '.*')
```

Comment:
Could you explain what's happening in default_alt and why it's necessary?

Reply:
Ah, sorry, I put a comment on that but it seems like a subsequent push resolved it 🤦‍♂️ 

Rails appears to have accidentally removed the default alt in v5.2.1 (I think they intended only to deprecate it). The logic here is a straight copy from what Rails used to do, and future-proofs us.

Reply:
Ohhh wow, that's silly of them to do that. Thanks for heads up and explaining!